vnext/ReactUWP/Modules/NativeUIManager.cpp

```diff
@@ -80,6 +80,25 @@ void NativeUIManager::DirtyYogaNode(int64_t tag)
   }
 }
 
+facebook::react::ShadowNode* NativeUIManager::getShadowNodeWithXamlRoot()
```

Comment:
>acebook::react::ShadowNode* [](start = 1, length = 27)

Consider changing this function to return the XamlRoot - as it stands it returns the first ShadowNode that has a XamlRoot, which makes it not very useful beyond asking it for the XamlRoot.

Reply:
Done